src/abi/index.ts

```diff
@@ -11,13 +10,16 @@ import contractAbiOperator from './AllowanceOperator.abi.json';
 import contractAbiDistributor from './DefaultDistributor.abi.json';
 import contractAbiAssessor from './DefaultAssessor.abi.json';
 import contractAbiRoot from './Root.abi.json';
+import contractAbiActions from './Actions.abi.json';
+import contractAbiProxy from './Proxy.abi.json';
+import contractAbiProxyRegistry from './ProxyRegistry.abi.json';
 import contractAbiTranche from './Tranche.abi.json';
 import contractAbiNFTData from './NftData.abi.json';
 
 import { ContractAbis } from '../types';
 
 export default {
-  COLLATERAL_NFT : contractAbiNFT,
+  COLLATERAL_NFT : contractAbiTitle,
```

Comment:
can you switch to use here. The merge is wrong 🙈The changes in Naxos are correct